### PR TITLE
jira playbook - now with actual jira

### PR DESCRIPTION
**Pull request by jphayes**

### CODE REVIEW

File: jira.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - role: liatrio.mount_persist_data
+      become: yes
+    - role: java_internal
+      become: yes
+    - role: postgres_prereqs
+      become: yes
+    - role: geerlingguy.postgresql
+      become: yes
+      ansible_distribution_version: 6.0.0
+      postgresql_version: 9.6
```
Comment (RobertKelly) [inline jira.yml:21]: Saw that Jira supported up to 9.3 officially but that's probably just out of date.  Assuming this works well.

**Review by RobertKelly — APPROVED** [review]
Reviewed and did not see any issues.

File: jira.yml (hunk 1 above)
Comment (jphayes) [reply jira.yml:21]: Newer versions support the newer version. :)

File: roles/java_internal/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  include_role:
+    name: geerlingguy.java
+  vars:
+    java_packages: java-1.8.0-openjdk-devel
+
+
+- name: change java alternative
+  alternatives:
+    name: java
+    path: /usr/lib/jvm/jre-1.8.0-openjdk.x86_64/bin/java
```
Comment (shanemacbride) [inline roles/java_internal/tasks/main.yml:12]: This is specific to Amazon Linux 1 as Amazon Linux 2 changes the directory name.

**Review by shanemacbride — APPROVED** [review]
Looks good to me! We just need to be sure to stick to the standard Amazon Linux AMI. Nice work.

**Pull request merged**